src/common/utils/gpo.h

```diff
@@ -305,4 +380,51 @@ namespace powertoys_gpo {
         return getConfiguredValue(POLICY_ALLOW_EXPERIMENTATION);
     }
 
+    inline gpo_rule_configured_t getRunPluginEnabledValue(std::string pluginID)
+    {     
+        if (pluginID == "" || pluginID == " ")
+        {
+            // this plugin id can't exist in the registry
+            return gpo_rule_configured_not_configured;
+        }
+
+        std::wstring plugin_id(pluginID.begin(), pluginID.end());
+        auto individual_plugin_setting = getPolicyListValue(POWER_LAUNCHER_INDIVIDUAL_PLUGIN_ENABLED_LIST_PATH, plugin_id);
+        
+        if (individual_plugin_setting.has_value())
+        {
+            string setting(individual_plugin_setting.value().begin(), individual_plugin_setting.value().end());
+            std::wstring force_disabled = L"0\0";
+            std::wstring force_enabled = L"1\0";
+            std::wstring user_takes_control = L"2\0";
+
+            auto var = (setting == L"0");
+
+            if (var)
+            {
+                // force disabled
+                return gpo_rule_configured_disabled;
+            }
+            else if (setting.compare(force_enabled) == 0)
+            {
+                // force enabled
+                return gpo_rule_configured_enabled;
+            }
+            else if (setting.compare(user_takes_control) == 0)
```

Comment:
@jaimecbernardo
I can read the registry without fixed array size now. But I can't compare the individual_plugin_setting which is now a std::wstring anymore. I tried almost everything I can find on the web. Can you please help.

Reply:
@htcfreek ,
My idea will be to try to debug into here and see what's up.
Can you please provide with some instructions to set up how you're testing this and then what's the expected behavior and what's happening? Thank you :)

Reply:
@jaimecbernardo
I need to push a code fix first. 

But what I need to do is comparing the content of two `std:wstring` vars or one `std:wstring` and one `std:string`.

If I do `var1 == var2` it compares the pointer or the length but not the character string itself. It was working in the past with two `std:string` vars.

Reply:
I have pushed a code change. We should continue discussion on changed code.